packages/sdk-cli/src/api/developerRelay.ts

```diff
@@ -54,6 +66,13 @@ export async function connect(hostID: string) {
 
 export async function hosts() {
   const response = await apiFetch('1/user/-/developer-relay/hosts.json')
+    .then(async (response: Response) => {
+      if (response.ok || response.status !== 403) return response;
+
+      const errorObj = decode(Hosts403Response)(await assertJSON()(response));
```

Comment:
Eating the response if it is not JSON of the right shape is going to make debugging unexpected error responses that much more frustrating for us. Fall back to the raw/pretty-printed JSON (or the response body as a string) if there is a decode error?